src/main/java/edu/unc/lib/boxc/migration/cdm/services/RedirectMappingIndexService.java

```diff
@@ -83,7 +83,11 @@ public void indexMapping() {
 
                 PreparedStatement preparedStatement = conn.prepareStatement(INSERT_STATEMENT);
                 preparedStatement.setString(1, cdm_collection_id);
-                preparedStatement.setString(2, cdm_object_id);
+                if (cdm_object_id.isEmpty()) {
```

Comment:
Its sort of not related to this PR, but I'm realizing this class has a bunch of variables using snake case instead of camel case. We could address that now, or do so in a follow up PR.

Reply:
ah i always get confused on which is proper format for which language. so variable names are camelCase?